profiler/src/test/java/com/splunk/opentelemetry/profiler/snapshot/SpecialTraceIds.java

```diff
@@ -0,0 +1,309 @@
+          Map.entry(-46, "017f8f890dba70b2c56e47b08c27cb9d"),
+          Map.entry(-45, "0d5da5aca4955083c684adb3de91969a"),
+          Map.entry(-44, "743e7d27b92ffba5c8b7cb2def891d59"),
+          Map.entry(-43, "c36412fff10e140fc92d4149555b7c2a"),
+          Map.entry(-42, "e2d3890b3fa2a0bdcadea6feeae1f717"),
+          Map.entry(-41, "0c9ebea92c46d291ccb12117443c2f7d"),
+          Map.entry(-40, "b60a11472cfde765cdda0285fc439bb0"),
+          Map.entry(-39, "4e4b3b2a775c874acf362b1ba9f36007"),
+          Map.entry(-38, "d1602621bee45ca7cf79c290095a578b"),
+          Map.entry(-37, "643160d634c8c180d167141416329b9d"),
+          Map.entry(-36, "b2636ad5ca821815d238a6d1ab614a98"),
+          Map.entry(-35, "840fa2f1cf3e9b15d3515df1073202f9"),
+          Map.entry(-34, "e3ff2c241deb1e5fd4b288e93867ceb5"),
+          Map.entry(-33, "94a292c490a7a6ffd6a921461f1d9303"),
+          Map.entry(-32, "11c7c62fb1ec3b71d75431a433a89b0c"),
+          Map.entry(-31, "754b92cc86b4413dd87f8396d1f5ca0d"),
+          Map.entry(-30, "0500ccb0a253074fd9b5f3fe67ad927b"),
+          Map.entry(-29, "7565828039676d19db78b956a3dbb7af"),
+          Map.entry(-28, "fafde45b6f3810c0dcec32328641ca87"),
+          Map.entry(-27, "9417f8ce5f3c9e22de1e497877b42c1c"),
+          Map.entry(-26, "49f42681ae2862f3df13a446fc8c0ad6"),
+          Map.entry(-25, "c13379b3753b3d9ae12089abced8594d"),
+          Map.entry(-24, "33071b9d7fa3f393e2720b88c55fcde9"),
+          Map.entry(-23, "e1d19a12b28045b8e33e35f525f0a0e6"),
+          Map.entry(-22, "61b0b41639df6abde4092c04cc065c8e"),
+          Map.entry(-21, "7821ebdaae0c7091e5c4d44018a77dc1"),
+          Map.entry(-20, "2744ccdafff26bd8e7681ccffd751eb5"),
+          Map.entry(-19, "80be4dc8e9588b65e833cb726d88cb9a"),
+          Map.entry(-18, "9612374da2e71d39e9f3ec1e650e677c"),
+          Map.entry(-17, "6223f180c35f8306ea50cdd2322192d5"),
+          Map.entry(-16, "fee2ec65b2283b4beccb4e2172515663"),
+          Map.entry(-15, "f868c9c00be1232fedc7dbcd2018ebc6"),
+          Map.entry(-14, "99541c86b4243423eed88e9495bf58aa"),
+          Map.entry(-13, "d2c1fcd29d071826f056c34335ccada3"),
+          Map.entry(-12, "fd7833d285c54bf1f1d19a57596dd046"),
+          Map.entry(-11, "fff195e8ba1cef66f2efe44045ea4100"),
+          Map.entry(-10, "4f022301835d6f58f4087c79d1f149bd"),
+          Map.entry(-9, "1022610fdec29ffef4f85a3ce56bbdc7"),
+          Map.entry(-8, "c44f365084aab186f68841e3e1208b38"),
+          Map.entry(-7, "ea6cbd5179d37f3ff7d0afe2a209fa68"),
+          Map.entry(-6, "48037208dec71569f8cb0b67c5e01381"),
+          Map.entry(-5, "986b203a02447d5df9bb619e3666421d"),
+          Map.entry(-4, "947035a119bd5455fb861c001125dc25"),
+          Map.entry(-3, "4cd88ee12d42abe5fc417cca5a01baac"),
+          Map.entry(-2, "436cdf1964184276fdbce3f148296994"),
+          Map.entry(-1, "d3d251f275071212fec6ac6aa3e880fa"),
+          Map.entry(0, "a28341e910992ea0fedb40ef4a39b11a"),
```

Comment:
This is a bit weird, I'd expect the value at 0 percentile to be never sampled. The second long in the trace id should be `Long.MIN_VALUE`. Here the 0 range seems to be the same as the -1 range.